database/CoreDatabaseManager.java

```diff
@@ -50,6 +50,8 @@ public class CoreDatabaseManager implements TypeDB.DatabaseManager {
 
     protected static final String RESERVED_NAME_PREFIX = "_";
 
+    protected static final int DIAGNOSTICS_SEND_PERIOD_MINUTES = 1;
```

Comment:
This update of database metrics is initiated by the `DatabaseManager`. I didn't want to make it more frequent in production, so 1 minute sounds like a balanced value.